setup.bat

```diff
@@ -6,3 +6,9 @@ venv\Scripts\python -m pip install --upgrade pip setuptools
 venv\Scripts\python -m pip install torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html
 venv\Scripts\python -m pip install -r requirements-win.txt
 cd ..
+cd rg_production
+python -m prodVenv
```

Comment:
why prodvenv and not simply venv, this is already inside the rg_production so can be the same directory and the venv is even already .gitignored inside rg_production repo not the prodVevn